### ISSUE TITLE
Failed to play some videos while use HttpProxyCacheServer

### ISSUE THREAD

**Issue by jerryisbusy** [danikula/AndroidVideoCache#121]
Origin url is http://ohwy83emy.bkt.clouddn.com/app_vedio/2017/03/08/1488947409951/1488947409951.mp4 which can be played through Chrome browser.

And in my application, wrapped url would like this http://127.0.0.1:36541/http://ohwy83emy.bkt.clouddn.com/app_vedio/2017/03/08/1488947409951/1488947409951.mp4 .

With IJKPlayer i can play with origin url, but cannot play with wrapped url.
I have did some tests, donwload from both two urls. The files are different.
The md5sum text of file from origin url is: 98fdf6fd427f78028445d2a857c9aadd,
however, the md5sum text  of file from wrapped url would change sometimes, but never equal to 98fdf6fd427f78028445d2a857c9aadd. And the file downloaded from wrapped url cannot be played on my computer. It seems like the file was broken. 
Well, both the size of two files are equal to 438140 bytes.

**Comment by XWZack** [comment]
run into same problem here,have u solved it?

**Comment by danikula** [comment]
@jerryisbusy , @XWZack do you try to use `exoPlayer` or `MediaPlayer`? Do you try to reproduce this issue with `sample` app? I tried play [your video](http://ohwy83emy.bkt.clouddn.com/app_vedio/2017/03/08/1488947409951/1488947409951.mp4) in `sample` app and didn't notice any problem. Can you make a super-duper simple app to give me chance reproduce your problem?

**Comment by jerryisbusy** [comment]
Sorry for late.
I use this library to play video https://github.com/Bilibili/ijkplayer.
So where can i find your sample app? I'd like to try it myself.
May be something's wrong with my project, I'm going to review my code now.

**Comment by danikula** [comment]
> So where can i find your sample app

I mean this ['sample' app](https://github.com/danikula/AndroidVideoCache/tree/master/sample)

**Comment by XWZack** [comment]
Sorry,I think its not your problem,i tried play my video in your sample app,and it's totally ok.

**Comment by danikula** [comment]
Problem may occur if use `ijkplayer` with `AndroidVideoCache`. Can you check it?

**Comment by XWZack** [comment]
Exactly,i use ijkplayer in my app. The weird thing is, it just occurs in some phone types.
so here is my video's url:
http://api.video.tianya.cn/v/play/20170315184959036f4702182 
and after being mapped:
http://127.0.0.1:38182/http%3A%2F%2Fapi.video.tianya.cn%2Fv%2Fplay%2F20170315184959036f4702182

**Comment by danikula** [comment]
@XWZack  can you fork my project and modify `sample` app to give me chance reproduce the problem?
